examples/compiled/rect_heatmap.vg.json

```diff
@@ -179,11 +179,6 @@
         },
         "axisY": {
             "minExtent": 30
-        },
```

Comment:
Sure, but I need an example to demonstrate what is our default ramp.  So you can either create a new example or tell Jeff to change the default.